packages/opentelemetry-tracer-basic/test/export/BatchSpanProcessor.test.ts

```diff
@@ -17,23 +17,10 @@
 import * as assert from 'assert';
 import { BatchSpanProcessor } from '../../src/export/BatchSpanProcessor';
 import { Span, BasicTracer } from '../../src';
-import { SpanExporter } from '../../src/export/SpanExporter';
-import { ReadableSpan } from '../../src/export/ReadableSpan';
+import { InMemorySpanExporter } from '../../src/export/InMemorySpanExporter';
 import { NoopScopeManager } from '@opentelemetry/scope-base';
 import { NEVER_SAMPLER, ALWAYS_SAMPLER, NoopLogger } from '@opentelemetry/core';
 
-// @todo: replace TestExporter with InMemorySpanExporter (pull/234)
-class TestExporter implements SpanExporter {
-  spansDataList: ReadableSpan[] = [];
-  export(spans: ReadableSpan[]): void {
-    this.spansDataList.push(...spans);
-  }
-
-  shutdown(): void {
-    this.spansDataList = [];
-  }
-}
-
 function createSampledSpan(spanName: string): Span {
   const tracer = new BasicTracer({
     scopeManager: new NoopScopeManager(),
```

Comment:
nit: I think it's not needed anymore, WDYT ?

Reply:
Sure, will do this in another PR.